src/Repository/CommitteeMembershipRepository.php

```diff
@@ -417,4 +417,21 @@ public function findCommitteesUuid(array $criteria): array
             return $uuid->toString();
         }, array_column($qb->getQuery()->getArrayResult(), 'uuid'));
     }
+
+    public function findCommitteeByPrivilege(Adherent $adherent, array $privilege): array
```

Comment:
Can you rename this method to `findCommitteesByPrivilege` ?

Reply:
@thomasbisignani done